add login html elements

kerolosNabil247 opened the pull request.

kerolosNabil247 committed:
login html elements

kerolosNabil247 committed:
feat(html): add login html elements

Added the following login html elements:
  - user name
  - password
  - login button

Closes #6

AAEmara commented on log_in.html:
```diff
@@ -6,6 +6,15 @@
+            <h1 class="title" id="title">Login page</h1>
+            <input type="text" class="nameText" id="name" placeholder="User Name"><br>
+            <input type="password" class="passwordText" id="password" placeholder="Password"><br>
+            <p class="passFaild" id="WrongPass">Please enter your correct password</p>
```
```suggestion
            <p class="passFailed" id="wrongPass">Please enter your correct password</p>
```

Please change those typos and inconsistent naming convention formats.

AAEmara commented on log_in.html:
```diff
@@ -6,6 +6,15 @@
+    <img src="" alt="" class="backgroundImg">
+    <div class="container">
+        <div class="login-card">
+            <h1 class="title" id="title">Login page</h1>
```
```suggestion
            <h1 class="title" id="title">WindyX</h1>
            <!-- Add the greeting message "Welcome Back" element referenced in the design.-->
```
Please change the title according to the design, while also adding the "Welcome Back" element.
![Login Page Design](https://github.com/user-attachments/assets/a75043c0-3cc1-4bc3-a127-63f01ec1d4e9)

AAEmara commented on log_in.html:
```diff
@@ -6,6 +6,15 @@
 </head>
 <body>
-    
+    <img src="" alt="" class="backgroundImg">
```
Please, add the following image background according to the design.
![Login Page Background Image](https://github.com/user-attachments/assets/f60aeb02-9129-430b-ae38-5a497c6c350a)

AAEmara commented on log_in.html:
```diff
@@ -6,6 +6,15 @@
     <title>Log in</title>
```
```suggestion
    <title>WindyX | Log in</title>
```

AAEmara reviewed (changes requested):
Please review the suggested comments and modify them to merge into the main branch.

Thank you.

kerolosNabil247 committed:
add bg image

kerolosNabil247 committed:
fix(html): add the missing elements and do the
            suggestions changes

    Added the following :
      -folder imgs containing the bg image
      - h1 element (WindyX)
    Changes the following elements in login html:
      - title from (Log in) to (WindyX | Log in)
      - img tag (add the path to the bg image)
      - greeting from (Log in) to (Welcome Back !)
      - change the class (passFaild) to (passFailed)
      - change the id (WrongPass) to (wrongPass)

    Closes #6

AAEmara reviewed (approved):
Congratulations! Changes have been adhering to the requested changes.
